src/twisted/conch/ssh/keys.py

```diff
@@ -32,7 +33,6 @@
 from twisted.conch.ssh.common import int_to_bytes
 from twisted.python import randbytes
 from twisted.python.compat import iterbytes, nativeString
-from twisted.python.constants import NamedConstant, Names
```

Comment:
thanks for the clenanup.

just a general note.

these code changes should have been done in the PR that added the deprecation

using the deprecated code will always have a small performance penalty due to the deprecation warning wrapping code.